docs/artifact_repositories/nuget.md

```diff
@@ -0,0 +1,100 @@
+# Phylum NuGet Registry
+
+The Phylum NuGet registry is based on NuGet's [Server API].
+
+[Server API]: https://learn.microsoft.com/en-us/nuget/api/overview
+
+## Configuration
```

Comment:
Should we also have a section for how to configure Visual Studio? It has a graphical configuration for this.

Reply:
We probably should, but I have no good way to test that right now. So I think it makes sense to just document the basics first, then do the Visual Studio stuff separately since it will require more work and testing.